projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -703,3 +714,12 @@ def __verify_payload_format(self, payload_dict: dict):
                     resolvable_by=ResolvableBy.USER_ERROR,
                 )
             )
+
+    @staticmethod
+    def __object_is_data_frame(obj: Union[dict, "pandas.DataFrame"]) -> bool:
+        try:
+            import pandas
+
+            return isinstance(obj, pandas.DataFrame)
+        except ImportError:
+            return False
```

Comment:
put a log.debug.

 It's a good idea to put debug message when ignoring an exception. It may give a hint to someone when debugging.